extensions/cli/theia/src/main/java/mil/nga/giat/geowave/format/theia/TheiaDownloadCommandLineOptions.java

```diff
@@ -0,0 +1,52 @@
+/*******************************************************************************
+ * Copyright (c) 2013-2018 Contributors to the Eclipse Foundation
+ * 
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License,
+ * Version 2.0 which accompanies this distribution and is available at
+ * http://www.apache.org/licenses/LICENSE-2.0.txt
+ ******************************************************************************/
+package mil.nga.giat.geowave.format.theia;
+
+import com.beust.jcommander.Parameter;
+
+public class TheiaDownloadCommandLineOptions
+{
+	@Parameter(names = "--userident", description = "email address to authentificate when downloading Theia imagery.")
+	private String userIdent;
+	@Parameter(names = "--password", description = "Password to authentificate when downloading Theia imagery.")
```

Comment:
adding required=true will help with CLI validation: `@Parameter(names = "--password", description = "Password to authentificate when downloading Theia imagery.", required = true)`